package.json

```
@@ -75,7 +75,7 @@
   },
   "peerDependencies": {
     "@openedx/frontend-build": ">= 13.0.15",
-    "@openedx/paragon": "^21.5.7",
+    "@openedx/paragon": ">= 21.5.7",
```

are we happy with just a `>=` here? before the switch from `@edx/paragon` to `@openedx/paragon` we had

https://github.com/openedx/frontend-platform/blob/b608ccdb8fff78658b7e6a49afea89a1c91a46c5/package.json#L78

maybe instead we want this to be

```suggestion
    "@openedx/paragon": ">= 21.5.7 < 23.0.0",
```

I don't have strong feelings one way or the other, just wanted to note that this is a departure from patterns we were using previously

---

Yeah, it would be safer to keep the semver range explicit (with a max version specified); I ended up going with the pattern of `@openedx/frontend-build` above, but agree we should de-risk future Paragon upgrades by making new major versions explicit with the tradeoff being it requires a maintenance a step to bump the version each time Paragon releases a new breaking change (not a big deal, IMO).

For example, if the `Button` component props API were to change in a v23 release and breaks frontend-platform's usage of it, we would want to prevent that by restricting the max version that can be supported.

I will update.